src/olympia/reviewers/utils.py

```diff
@@ -1103,7 +1105,7 @@ def log_sandbox_message(self):
 
 class ReviewUnlisted(ReviewBase):
 
-    def process_public(self):
+    def approve_latest_version(self):
```

Comment:
Not related to this patch, I filed this issue earlier https://github.com/mozilla/addons-server/issues/14943 and this renaming makes me think that this is where we want to fix the problem.

Reply:
I think for #14943 we'll likely need to have a method in the `ReviewBase` class that does it and make sure it's called for a bunch of actions.